js/page-specific/citysourced-viewer.js

```diff
@@ -3,27 +3,136 @@ namespacer("baltimoreCounty.pageSpecific");
 baltimoreCounty.pageSpecific.citySourcedViewer = (function($, querystringer, maps, undefined) {
 	'use strict';
 
-	var init = function() {
+	var getNearbyData = function(settings, callback) {
+		
+
+			$.ajax('//testservices.baltimorecountymd.gov/api/citysourced/getreportsbylatlng', {
+			//$.ajax('//ba224964:1000/api/citysourced/getreportsbylatlng', {
+				method: 'POST',
+				data: settings
+			})
+				.done(function(data) {
+					if (callback)
+						callback(data);
+				})
+				.fail(function(error) {
+
+				});
+		},
+
+		/**
+		 * Descending date sort.
+		 */
+		commentDateComparer = function(a, b) {
+			var aMoment = moment(a.Created),
+				bMoment = moment(b.Created);
+
+			if (aMoment.isAfter(bMoment)) 
+				return -1;
+			else 
+				return 1;
+		},
+
+		/**
+		 * Hides all but the top `numberToShow` comments in the `$target` list.
+		 */
+		hideMostComments = function($target, commentData, numberToShow) {
+			if (commentData && commentData.length > numberToShow) {
+				$target.find('li').not(function(index, element) {
+					return index <= numberToShow - 1;
+				}).hide();
+				$target.after('<p><a href="javascript:;" id="show-comments" aria-role="button">Load ' + (commentData.length-numberToShow) +' more comments</a></p>');
+				$('#show-comments').on('click', function() {
+					$target.find('li').slideDown(500);
+					$(this).hide();
+				});	
+			}
+		},
+
+		/**
+		 * Removes seconds from the date strings of a data item.
+		 */
+		processTimeForViewerDataItem = function(data) {
+
+			if (data.DateCreated)
```

Comment:
should use brackets, since it's multi line conditional (i might have made this up)